cmd/domasimu/main.go

```diff
@@ -6,14 +6,17 @@
 package main
 
 import (
+	"context"
 	"flag"
 	"fmt"
 	"os"
 	"path/filepath"
+	"strconv"
 	"strings"
 
 	"github.com/BurntSushi/toml"
-	"github.com/pearkes/dnsimple"
+	"github.com/dnsimple/dnsimple-go/dnsimple"
```

Comment:
Will you please update the .travis.yml to reflect this package change so that tests will fetch the right package?